src/features/bulk/FileViewer/FileViewer.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ComponentProps } from 'react';
+import { screen, act } from '@testing-library/react';
+import { DocumentViewer } from '@amsterdam/bmi-component-library';
+
+import { render } from '../../../tests/utils/testUtils';
+import { mockComponentProps, mocked } from '../../../tests/helpers';
+import { files as filesMock, fields as fieldsMock } from '../bulk/__stubs__';
+import { getDocumentViewUrlMock, onChangeMock } from '../bulk/__mocks__/bulk';
+import FileViewer from './FileViewer';
+
+const defaultProps = {
+	file: filesMock[0],
+	getDocumentViewUrl: getDocumentViewUrlMock,
+	onChange: onChangeMock,
+};
+
+jest.mock('@amsterdam/bmi-component-library', () => ({
+	...jest.requireActual('@amsterdam/bmi-component-library'),
+	DocumentViewer: jest.fn().mockImplementation(() => <div data-testid="document-viewer" />),
+}));
+
+// Mock IndividualFieldsForm because <Form> doesn't return anything
+jest.mock('./IndividualFieldsForm', () => {
+	return jest.fn().mockImplementation(({ fields }) => {
+		return (
+			<div data-testid="form">
+				{fields.map((field: any) => (
+					<span key={field.id}>{field.label}</span>
+				))}
+			</div>
+		);
+	});
+});
+
+afterEach(() => {
+	jest.restoreAllMocks();
+});
+
+describe('<FileViewer />', () => {
+	test('Default fields are rendered', async () => {
+		const store = {
```

Comment:
This can be DRYed up a lot by moving the render logic in an async beforeEach block no?

Reply:
You would think, I've tried this a few moments ago, but moving the render to a beforeEach didn't work. I did remove some duplication by moving the store object.